src/components/availability-panel/drag-drop-context/DraggableComponent.tsx

```diff
@@ -38,6 +37,13 @@ const DraggableComponent: FC<draggableProps> = ({
     title = '',
 }) => {
     const draggableIds = useContext(disableDrag);
+    const { data: userResponse } = useGetUsersByUsernameQuery({
+        searchString: draggableId,
+        size: 1,
+    });
+    const draggableUserImage: string =
```

Comment:
Suggestion: Renamed it to `draggableUserImageURL`.

Reply:
Changes done